README.md

```diff
@@ -42,6 +42,7 @@ size).
 | opts.height | <code>integer</code> | output height |
 | opts.loop | <code>boolean</code> | whether to loop the source clip (defaults to true) |
 | opts.start | <code>float</code> | seek to this time before starting. Must be less |
+| opts.inputDuration | <code>float</code> | if set we can use this to make sure we don't attempt to start after the end of the input file (ie. we can loop or repeat the last frame correctly rather than seg faulting). |
```

Comment:
@aeh Wondering whether we can just make this mandatory, and push this out as a breaking change.  Is there ever a case where we won't provide this?